lib/auth/join.go

```diff
@@ -389,7 +393,32 @@ func (a *Server) generateCertsBot(
 		if err != nil {
 			log.WithError(err).Warn("Unable to encode join attributes for audit event.")
 		}
+
+		auth.Metadata, err = structpb.NewStruct(attributes)
```

Comment:
Also not certain of what metadata means per the RFD - I assume it's this (join method claims)?

Reply:
Yup - this is what I meant 😅